fashion-messaging-app-ui/src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import ReactDOM from "react-dom/client"
 import App from "./App.jsx"
-import "./index.css"
+import "./global.css"
```

Comment:
I changed this because it has all my color variables for my css so it has more specific naming convention